qwik-graphql-tailwind/.storybook/preview.ts

```diff
@@ -1,5 +1,5 @@
 // Based on https://github.com/BuilderIO/qwik/discussions/787#discussioncomment-3715103
-import { JSXNode, useContextProvider } from '@builder.io/qwik';
```

Comment:
this file shouldn't be touched, can you revert this one

Reply:
that import was unused, do we still need that `useContextProvider`?